test/SparqlExpressionTest.cpp

```diff
@@ -704,6 +704,50 @@ TEST(SparqlExpression, substr) {
               IdOrLiteralOrIri{lit("bye")});
 }
 
+// _____________________________________________________________________________________
+TEST(SparqlExpression, strIriDtTagged) {
+  auto U = Id::makeUndefined();
+  auto checkStrIriTag =
+      std::bind_front(testNaryExpression, &makeStrIriDtExpression);
+  checkStrIriTag(IdOrLiteralOrIriVec{lit(
+                     "123", "^^<http://www.w3.org/2001/XMLSchema#integer>")},
+                 IdOrLiteralOrIriVec{lit("123")},
+                 IdOrLiteralOrIriVec{
+                     iriref("<http://www.w3.org/2001/XMLSchema#integer>")});
```

Comment:
Those should actually lead to a TripleComponent with type Int,
But this can be fixed once we have a function which another student is about to write).
We can then backport this to this PR.